tests/test.linter.js

```diff
@@ -183,6 +183,25 @@ describe('Linter', function() {
       });
   });
 
+  it('Eslint ignore patterns and .eslintignorerc should be ignored', () => {
+    // Verify https://github.com/mozilla/addons-linter/issues/1288 is fixed
+    var addonLinter = new Linter({_: [
+      'tests/fixtures/webextension_node_modules_bower']});
+
+    // Stub print to prevent output.
+    addonLinter.print = sinon.stub();
+
+    return addonLinter.scan()
+      .then(() => {
+        assert.deepEqual(addonLinter.collector.scannedFiles, [
```

Comment:
This looks good but as mentioned above (https://github.com/mozilla/addons-linter/pull/1296/files#r118443364) this is only showing it was scanned at all and not explicitly by eslint from what I can tell. Might be worth putting a JS error back in the `bar.js` and `foo.js` just to make sure they were scanned by eslint.

That and/or categorise the files scanned, but if you wanted to do that in a separate issue and just check for eslint errors for now to get this patch through that works for me.